cmd/livepeer/livepeer.go

```diff
@@ -268,10 +270,8 @@ func main() {
 		n.NodeType = core.TranscoderNode
 	} else if *broadcaster {
 		n.NodeType = core.BroadcasterNode
-	} else if *redeemer {
-		n.NodeType = core.RedeemerNode
 	} else {
-		glog.Fatalf("Node type not set; must be one of -broadcaster, -transcoder or -orchestrator")
+		glog.Info("Node type not set with -broadcaster, -transcoder, -orchestrator or -redeemer")
```

Comment:
It does seem like allowing a "default" node type that basically does nothing is a bit of a usability regression. I understand that we didn't want to enumerate a bunch of NodeTypes (and I think we probably should rework that mechanism entirely), but in the past, we've failed fast if the user didn't specify the mode they were in.

Right now, we don't fail fast - we just sit there with services disabled - and I think that's going to confuse some people. This log message might help a bit, but it's not immediately obvious, because it's sandwiched in between a bunch of other messages:

```
I0702 12:41:17.288835    6911 livepeer.go:215] ***Livepeer is running on the offchain network***
I0702 12:41:17.290573    6911 db.go:376] Initialized DB node
I0702 12:41:17.290629    6911 livepeer.go:279] Node type not set with -broadcaster, -transcoder, -orchestrator or -redeemer
I0702 12:41:17.290662    6911 livepeer.go:320] ***Livepeer is in off-chain mode***
I0702 12:41:17.291110    6911 livepeer.go:871] Current ManifestID will be available over 
I0702 12:41:17.291175    6911 mediaserver.go:137] Transcode Job Type: [{144p 100000 0 0 256x144  0 0 0s} {240p 150000 0 0 320x240  0 0 0s}]
I0702 12:41:17.291239    6911 webserver.go:70] CLI server listening on 127.0.0.1:7935
```